lib/Client/ClientSpan.php

```diff
@@ -95,6 +95,13 @@ public function setEndUserId($id) {
         return $this;
     }
 
+    public function setTags($fields) {
+        foreach ($fields as $item) {
```

Comment:
So if I'm reading this correctly, your call to `setTags` would look like:

```
$span->setTags([['tag1', 'value1'], ['tag2', 'value2'], ...]);
```

It looks to me like the intended usage of the method is:
```
$span->setTags(['tag1' => 'value1', 'tag2' => 'value2', ...]);
```

If that's the case, this loop would need to look like:

```
foreach ($fields as $key => $value) {
```

If I'm wrong, feel free to ignore this.

Reply:
I kinda guessed as to how people were passing in multi-value tags, but based on the docs, it looked like you're expected to call `startSpan` like this: `$tracer->startSpan("test", array('tags' => array( 'foo' => 'bar', 'baz' => 'quuz')));` Is that not how you'd do it?

Reply:
That's exactly how we tried to use it.

Reply:
Ah, then I think the way I've got it implemented now works then, based on the tests I added?

Reply:
🤦‍♂ ah, i get it now. yeah, fixing this now, thanks!

Reply:
This looks like I'd expect now!